### PR TITLE
Disable tests affected by #3147 except on Linux in Release mode.

### PR DESCRIPTION
**Pull request by david-german-tri**
@RussTedrake, @siyuanfeng-tri FYI

---
This change is [Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/3148)

### CODE REVIEW

**Comment by david-german-tri** [comment]
Linux debug: https://drake-jenkins.csail.mit.edu/view/Experimental/job/linux-gcc-experimental-matlab/72/

Linux release: https://drake-jenkins.csail.mit.edu/view/Experimental/job/linux-gcc-experimental-matlab/73/

**Comment by david-german-tri** [comment]
Mac debug: https://drake-jenkins.csail.mit.edu/view/Experimental/job/mac-clang-experimental-matlab/67/

Mac release: https://drake-jenkins.csail.mit.edu/view/Experimental/job/mac-clang-experimental-matlab/66/

**Comment by david-german-tri** [comment]
Sigh, this approach has apparently disabled the tests in Linux/Release also.

**Comment by david-german-tri** [comment]
Linux release, now with more STREQUAL: https://drake-jenkins.csail.mit.edu/view/Experimental/job/linux-gcc-experimental-matlab/75/

Mac release, just to make sure we didn't regress: https://drake-jenkins.csail.mit.edu/view/Experimental/job/mac-clang-experimental-matlab/68/

**Comment by david-german-tri** [comment]
@jamiesnape, @BetsyMcPhail for feature review

File: drake/examples/Atlas/CMakeLists.txt [hunk 1]
```diff
@@ -32,8 +32,11 @@ add_matlab_test(NAME examples/Atlas/runCOMFixedPointSearch COMMAND runCOMFixedPo
 add_matlab_test(NAME examples/Atlas/runRobotiqPDControl COMMAND runRobotiqPDControl)
 
 if(LONG_RUNNING_TESTS)
-  add_matlab_test(NAME examples/Atlas/runAtlasBalancingPerturb COMMAND runAtlasBalancingPerturb PROPERTIES TIMEOUT 1500 REQUIRES gurobi)
-  add_matlab_test(NAME examples/Atlas/runAtlasManip COMMAND runAtlasManip PROPERTIES TIMEOUT 1500 REQUIRES gurobi)
+
+  if (${CMAKE_BUILD_TYPE} STREQUAL "Release" AND NOT APPLE) # FIXME: see #3147.
```
Comment (jamiesnape) [inline drake/examples/Atlas/CMakeLists.txt]: `NOT CMAKE_BUILD_TYPE STREQUAL "Debug"`? 
`UNIX AND NOT APPLE` or `NOT WIN32`?

**Comment by jamiesnape** [comment]
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/3148)*

**Comment by david-german-tri** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[drake/examples/Atlas/CMakeLists.txt, line 36 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/3148#-KPFBWG9SBwrPFpoDzPA-r1-36:-KPJNsldkjqOP7W7Y5KM:b-896fix) ([raw file](https://github.com/robotlocomotion/drake/blob/a95365eb0f222992c3f127f731818aad34f50cfa/drake/examples/Atlas/CMakeLists.txt#L36)):*
<details><summary><i>Previously, jamiesnape (Jamie Snape) wrote…</i></summary>
> `NOT CMAKE_BUILD_TYPE STREQUAL "Debug"`? 
> 
> `UNIX AND NOT APPLE` or `NOT WIN32`?
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/3148)*

**Comment by david-german-tri** [comment]
One more time, Linux release: https://drake-jenkins.csail.mit.edu/view/Experimental/job/linux-gcc-experimental-matlab/78/

**Comment by jamiesnape** [comment]
:lgtm:

---

Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/3148#-:-KPJOEtRz32ovGi0IvTZ:bnfp4nl)*

**Comment by david-german-tri** [comment]
+@sammy-tri for platform review

---

Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/3148#-:-KPJQ00UnrJN1o7ayBFK:bq11ftp)*

**Comment by sammy-tri** [comment]
:lgtm:

---

Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/3148#-:-KPJivesaL7c3ZcqpGch:bnfp4nl)*

**Pull request merged**